libraries/abstractions/common_io/test/test_iot_i2c.c

```diff
@@ -126,6 +126,9 @@ TEST_TEAR_DOWN( TEST_IOT_I2C )
 void prvI2CCallback( IotI2COperationStatus_t xOpStatus,
                      void * pvParam )
 {
+    /* Silence the compiler. */
```

Comment:
Can you mention something like `Disable unused parameter warning.` instead to clearly mention the warning being silenced

Reply:
Sure.

Reply:
Yes please "Silence the compiler" is not a good comment